x-pack/plugins/security_solution/server/lib/timeline/saved_object/notes/saved_object.ts

```diff
@@ -87,35 +85,17 @@ export const persistNote = async ({
   note: BareNote;
   overrideOwner?: boolean;
 }): Promise<ResponseNote> => {
-  try {
-    if (noteId == null) {
-      return await createNote({
-        request,
-        noteId,
-        note,
-        overrideOwner,
-      });
-    }
-
-    // Update existing note
-    return await updateNote({ request, noteId, note, overrideOwner });
-  } catch (err) {
-    if (getOr(null, 'output.statusCode', err) === 403) {
-      const noteToReturn: Note = {
-        ...note,
-        noteId: uuidv1(),
-        version: '',
-        timelineId: '',
-        timelineVersion: '',
-      };
-      return {
-        code: 403,
-        message: err.message,
-        note: noteToReturn,
-      };
-    }
-    throw err;
+  if (noteId == null) {
+    return createNote({
+      request,
+      noteId,
+      note,
+      overrideOwner,
+    });
```

Comment:
One question regarding this. Why was exception clause removed. I see that neither parent not child of `createNote` is setting the appropriate status code. I can see `403` return code is wrong here but did you want to return default `500` status code when this kind of error occurs?